dts/bindings/interrupt-controller/snps,archs-idu-intc.yaml

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2019, synopsys
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: ARC-HS Interrupt Distribution Unit
+
+description: >
+    This binding describes the 2nd level interrupt controller can be used in
+    SMP configurations for dynamic IRQ routing, load balancing of
+    common/external IRQs towards core intc
+
+inherits:
+    !include base.yaml
```

Comment:
Update this to be:
`!include [interrupt-controller.yaml, base.yaml]`

Reply:
fixed